src/commands/lambda/README.md

```diff
@@ -145,6 +145,35 @@ Instead of supplying arguments, you can create a configuration file in your proj
     }
 }
 ```
+
+## Troubleshooting Datadog Lambda Monitoring
```

Comment:
```suggestion
## Troubleshoot Serverless Monitoring
```
Maybe?
Since https://docs.datadoghq.com/serverless/troubleshooting/
Maybe asking PMs or Support Team would be better.